src/Sylius/Bundle/CoreBundle/Resources/config/app/main.yml

```diff
@@ -13,7 +13,6 @@ twig:
 
 doctrine_cache:
     providers:
-        sylius_settings: %sylius.cache%
```

Comment:
I removed this because you can't be sure if the given `schemaAlias` is the only one. Since you now have settings resolvers it is not simply `namespace` anymore to make it unique. We could cache them by settings id now? But doesn't doctrine already provide us with an identity map and have this build in?